services/QuillLMS/app/helpers/vitally_teacher_stats_helper.rb

```diff
@@ -0,0 +1,63 @@
+module VitallyTeacherStatsHelper
```

Comment:
Helpers are usually designed for for views. This would make more sense in a app/models/concern